server/pbenchinacan/pod-nodata.yml

```diff
@@ -154,7 +154,7 @@ spec:
     imagePullPolicy: Always
     name: pbenchserver
     ports:
-      - containerPort: 80
+      - containerPort: 8080
```

Comment:
Interestingly, at line 132 we specify a `containerPort` but don't specify a `hostPort`, presumably because we want it to be the same...which leads me to suggest that we do the same thing here (and at line 64, and likewise in `server/pbenchinacan/pod.yml`).